akka-docs/src/main/paradox/scala/distributed-pub-sub.md

```diff
@@ -30,8 +30,12 @@ any other node.
 There a two different modes of message delivery, explained in the sections
 [Publish](#distributed-pub-sub-publish) and [Send](#distributed-pub-sub-send) below.
 
+@@@ div { .group-scala }
+
 A more comprehensive sample is available in the
-tutorial named [Akka Clustered PubSub with Scala!](https://github.com/typesafehub/activator-akka-clustering).
+tutorial named @scala[[Akka Clustered PubSub with Scala](http://dev.lightbend.com/start/?group=akka&project=akka-samples-cluster-scala)]@java[[Akka Clustered PubSub with Java](http://dev.lightbend.com/start/?group=akka&project=akka-samples-cluster-java)].
+
+@@@
```

Comment:
Hmm, looking more closely I'm afraid the `akka-samples-cluster` sample doesn't in fact contain a pubsub example... Sorry about the churn, perhaps it's best to revert that last change and merge it with the link to Activator (until we migrate that to techhub)